userspace/otpilot/src/main.rs

```diff
@@ -82,21 +84,13 @@ fn run() -> TockResult<()> {
         device_id: [0; 64],
     };
 
-    {
-        let mut idx : usize = 0;
-        for val in "v1.00".as_bytes() {
-            if idx > identity.version.len() { break; }
-            identity.version[idx] = *val;
-            idx = idx + 1;
-        }
-    }
+    let banner = "v1.00".as_bytes();
+    let max_len = min(identity.version.len(), banner.len());
```

Comment:
If banner is too long, this will silently drop the tail end.  Is that OK?

Reply:
Yeah, I'd say that's ok. We're limited by the size that Manticore gives us in those fields.

Reply:
We could print out an error message and continue ...